book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    @rentals << Rental.new(person, date, self)
+  end
```

Comment:
- [ ] Remember that in the Rental constructor, the rental adds itself to the `rentals` array so you don't need to add the rental here again.

- [ ] The order of the rental arguments should be maintained too. The Rental class expects parameters in this order: `book`, `person`, `date`. So the same should be replicated here.

  In essence this method should look like this:

  ```
  def add_rental(person, date)
    Rental.new(self, person, date)
  end
  ```